Using literal in FindOptions

Hello,
I'm creating a query with the code:
```typescript
public static getAllByVenueId(venueId) {
    return Deal.findAll({
      where: {venueId},
      attributes: {
        include: [
          [sequelize.literal('CURRENT_DATE > "expiryDate"'), "expired"],
          [sequelize.literal('CURRENT_TIMESTAMP < "promotedUntil"'), "promoted"]
        ],
        exclude: ["updatedAt", "createdAt"]
      }
    });
  }
```

When compiling I got the following typescript warnings:
```
server/src/models/Deal.ts(78,25): error TS2345: Argument of type '{ where: { venueId: any; }; attributes: { include: [literal, string][]; exclude: string[]; }; }' is not assignable to parameter of type 'IFindOptions'.
  Types of property 'attributes' are incompatible.
    Type '{ include: [literal, string][]; exclude: string[]; }' is not assignable to type '(string | [string, string] | fn | [fn, string] | cast | [cast, string])[] | { include?: (string |...'.
      Type '{ include: [literal, string][]; exclude: string[]; }' is not assignable to type '{ include?: (string | [string, string] | fn | [fn, string] | cast | [cast, string])[]; exclude?: ...'.
        Types of property 'include' are incompatible.
          Type '[literal, string][]' is not assignable to type '(string | [string, string] | fn | [fn, string] | cast | [cast, string])[]'.
            Type '[literal, string]' is not assignable to type 'string | [string, string] | fn | [fn, string] | cast | [cast, string]'.
              Type '[literal, string]' is not assignable to type '[cast, string]'.
                Type 'literal' is not assignable to type 'cast'.
                  Property 'type' is missing in type 'literal'.
11:55:04 - Compilation complete. Watching for file changes.
```
However the generated code works fine.

--- a/lib/interfaces/IFindOptions.ts
+++ b/lib/interfaces/IFindOptions.ts
@@ -13,7 +13,7 @@ export interface IFindOptions<T> extends LoggingOptions, SearchPathOptions {
   /**
    * A hash of attributes to describe your search. See above for examples.
    */
-  where?: WhereOptions<T> | where | fn | or | Array<col | and | or | string>;
+  where?: WhereOptions<T> | where | fn | literal | or | Array<col | literal | and | or | string>;
 
   /**
    * A list of the attributes that you want to select. To rename an attribute, you can pass an array, with

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sequelize-typescript",
-  "version": "0.6.5-beta.0",
+  "version": "0.6.5-beta.1",
   "description": "Decorators and some other extras for sequelize (v3 + v4)",
   "scripts": {
     "build": "tsc --project lib/models/v4 && tsc",

